app/views/unattended/provisioning_templates/host_init_config/host_init_config_default.erb

```diff
@@ -15,7 +15,7 @@ set -e
 # This script enrolls a host with Foreman.
 # Host: <%= @host.name %>
 
-<% if @host.puppetmaster.present? -%>
+<% if host_puppetserver.present? -%>
```

Comment:
Can we be consistent with `host_puppetserver.present?` vs `!host_puppetserver.blank?`